packages/veritone-react-common/src/components/Breadcrumbs/BreadcrumbElipsis.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Do NOT separate this component from the main Breadcrumb component. It is NOT complex enough to justify pulling it out into its own file.

You should only create separate files for components that have specialized logic tightly bound to them (and are reusable).

This component is only used once, is merely an icon, and a click func. There is nothing special about it. Please move this component into `index.js`.

Reply:
Done!